### DISCUSSION TITLE
[ UPDATE!!! ] Simulacra

### DISCUSSION THREAD

**Discussion by Jeyphr** [nzp-team/nzportable#858] [Map Releases]
### **WELCOME TO SIMULACRA**
_Simulated American town
located in a German viral
testing facility. Do you
have what it takes to
survive?_

### [>>> DOWNLOAD LINK <<<](https://archive.org/download/simulacra_202407/Simulacra%20V1.5.zip)
[PROJECT FILES](https://archive.org/details/simulacra_202407)
[VIDEO SHOWCASE](https://www.youtube.com/watch?v=8c4uVK3f_-A&ab_channel=Jeyphr)

SCREENSHOTS: 
---
![screenie1](https://github.com/user-attachments/assets/0f76a4d5-572d-427c-95ca-7de35fb36dc6)
![screenie2](https://github.com/user-attachments/assets/90f43e39-2c0b-4bb0-a460-a06f616f4176)
![screenie3](https://github.com/user-attachments/assets/f643dbc2-bf65-42b2-84a9-ce0b19b5a2fe)

---
### UPDATE LOG:
- Fixed a model error bug caused by not assigning a model in the func_ending entity
- Fixed a minor visual bug with the stairs above PHD Flopper.
- Applied basic optimizations to all the func_detail objects
- Fixed a zombie stuck spot on the wooden plank next to power
- Adjusted clipping brushes to ease navigation
- Made obtaining Double Tap II objectively more clear
- Removed the invalid geometry caused by the generators
- Used -wadinclude
- Cleaned up some of the bad geometry
- Updated the bus to be a func_detail
- Updated the roofs of the houses to be a func_detail
- Removed invalid func_wall entities

**Comment by MotoLegacy** [comment]
> Good luck getting it to run on a Psp.

Why do you say this? Screenshots make it look like it'll be great

**Reply by Jeyphr** [reply]
It's just a big map, that's all. Similar thing happened with that Town remaster someone made a while back.

**Reply by MotoLegacy** [reply]
Problem with that was the mapper really didn't care about invalid brushes or anything. That map won't even run on Nintendo Switch

**Comment by sbeeds** [comment]
![20240731_181541](https://github.com/user-attachments/assets/a7bef0f7-3e38-4107-894a-80d9f466ca8a)

Did you compile with -nowadtextures? I noticed you included a wad, but my game failed to load it. This could be an issue with my game, but it's a good idea to compile with that option enabled if you haven't

**Reply by Jeyphr** [reply]
What's -nowadtextures? I made the map in J.A.C.K and I can't seem to find that option.

**Reply by sbeeds** [reply]
This is an example of the command used in the 'compile' screen for the CSG tool. 

- "${WORK_DIR_PATH}/${MAP_BASE_NAME}.map" -nowadtextures

It includes the textures in the bsp so that you don't need to distribute a wad

**Reply by Jeyphr** [reply]
I also noticed that the texture that you're missing is blackNoise, which is a depracated texure. It should read nse_black. It's a mapping error. I applied the wrong texture.

**Reply by sbeeds** [reply]
ah, that makes sense. looking forward to the updated version!

**Reply by MotoLegacy** [reply]
Just making a note here that any non-PC platform cannot play this due to this issue :)

**Comment by bigjackson** [comment]
really cool map, but one thing i'd suggest is to fix the wall weapons so you don't have to be at an angle to buy them, but other than that this is a really good map.

**Reply by Jeyphr** [reply]
Thank you! Your feedback is greatly appreciated.

**Comment by SgtCookieBobble** [comment]
Found this surprisingly playable on the Vita! Lowest drop was about 22 FPS, but I do love the aesthetic.

**Comment by nonMat** [comment]
Oh wow this is great!!! The vibe is amazing and it's also quite hard! Love it!

**Reply by nonMat** [reply]
<img width="1920" height="1080" alt="immagine" src="https://github.com/user-attachments/assets/5e0d81c9-088e-4d81-aeaf-b1f4d5bf2968" />
So fun!

**Reply by Jeyphr** [reply]
Thanks for playing!

**Comment by hunterjj2012-prog** [comment]
there is a bug: idk if its wanted but on the bus the zombies slide down
<img width="315" height="415" alt="{7615DD49-5C10-4FD9-A069-B48F4475ABA1}" src="https://github.com/user-attachments/assets/7974240f-226b-4c00-b40a-5851647e7824" />
